internal/yunikorn/sync.go

```diff
@@ -224,16 +187,10 @@ func (s *Service) syncApplications(ctx context.Context) error {
 	return nil
 }
 
-func (s *Service) syncHistory(ctx context.Context) error {
-
-	appsHistory, err := s.client.GetAppsHistory(ctx)
-	if err != nil {
-		return fmt.Errorf("could not get apps history: %w", err)
-	}
-	containersHistory, err := s.client.GetContainersHistory(ctx)
-	if err != nil {
-		return fmt.Errorf("could not get containers history: %w", err)
-	}
+func (s *Service) syncHistory(
```

Comment:
We should probably split this into syncAppHistory and syncContainerHistory. is there a good reason why this method is handling multiple things?

Reply:
yeah, I was thinking about it, Actually was designed this way from the beginning so I just carried it forward.